{
  "affected_release": [
    {
      "product_name": "Red Hat Enterprise Linux 3",
      "release_date": "2009-07-27T00:00:00Z",
      "advisory": "RHSA-2009:1178",
      "package": "python-0:2.2.3-6.11",
      "cpe": "cpe:/o:redhat:enterprise_linux:3"
    },
    {
      "product_name": "Red Hat Enterprise Linux 4",
      "release_date": "2009-07-27T00:00:00Z",
      "advisory": "RHSA-2009:1177",
      "package": "python-0:2.3.4-14.7.el4_8.2",
      "cpe": "cpe:/o:redhat:enterprise_linux:4"
    },
    {
      "product_name": "Red Hat Enterprise Linux 5",
      "release_date": "2009-07-27T00:00:00Z",
      "advisory": "RHSA-2009:1176",
      "package": "python-0:2.4.3-24.el5_3.6",
      "cpe": "cpe:/o:redhat:enterprise_linux:5"
    }
  ],
  "package_state": null,
  "threat_severity": "Low",
  "public_date": "2008-06-02T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2008-3144 python: Potential integer underflow and overflow in the PyOS_vsnprintf C API function",
    "id": "455018",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455018"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "",
    "cvss3_scoring_vector": "",
    "status": ""
  },
  "iava": "",
  "cwe": "CWE-190",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2008-3144",
  "document_distribution": "",
  "details": [
    "Multiple integer overflows in the PyOS_vsnprintf function in Python/mysnprintf.c in Python 2.5.2 and earlier allow context-dependent attackers to cause a denial of service (memory corruption) or have unspecified other impact via crafted input to string formatting operations.  NOTE: the handling of certain integer values is also affected by related integer underflows and an off-by-one error."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2008-3144\nhttps://nvd.nist.gov/vuln/detail/CVE-2008-3144"
  ]
}